Typos docs (#1212)
temporalio/documentation: docs/application-development-guide.md

Calls to spawn [Activity Executions](/activities/#activity-execution) are written within a [Workflow Definition](/workflows/#workflow-definitions).
The call to spawn an Activity Execution generates the [ScheduleActivityTask](/references/commands/#scheduleactivitytask) Command.

> This results in the set of three [Activity Task](/tasks/#activity-task) related Events ([ActivityTaskScheduled](/references/events/#activitytaskscheduled), [ActivityTaskStarted](/references/events/#activitytaskstarted), and ActivityTask[Closed](/references/events/#activitytaskclosed)) in your Workflow Execution Event History.

Reviewer: ```suggestion
This results in the set of three [Activity Task](/tasks/#activity-task) related Events ([ActivityTaskScheduled](/references/events/#activitytaskscheduled), [ActivityTaskStarted](/references/events/#activitytaskstarted), and one of the Closed status Events) in your Workflow Execution Event History.
```

There is no ActivityTaskClosed Event - this wasn't meant to be a link
this was meant to be a variable placeholder for one of the Closed Event statuses.

However this change needs to be made within the source file, which I think is the `app-dev-context/spawning-activities.md` file -

#### Set Workflow Id

> While it is not required, we recommend providing your own [Workflow Id](/workflows/#workflow-id) that maps to a business process or business entity identifier, such as an order identifier or customer identifier.

Reviewer 2: ```suggestion
Although it's not required, we recommend providing your own [Workflow Id](/workflows/#workflow-id) that maps to a business process or business entity identifier, such as an order identifier or customer identifier.
```
"While" -> "Although" per "while" in https://developers.google.com/style/word-list#letter-w.

> Although it's not required, we recommend providing your own [Workflow Id](/workflows/#workflow-id) that maps to a business process or business entity identifier, such as an order identifier or customer identifier.

Reviewer: Again, this needs to be fixed in the source file -
